src/box/lua/connpool.lua

```diff
@@ -202,8 +271,10 @@ local function filter(opts)
         mode = '?string',
     })
     opts = opts or {}
-    if opts.mode ~= nil and opts.mode ~= 'ro' and opts.mode ~= 'rw' then
-        local msg = 'Expected nil, "ro" or "rw", got "%s"'
+    if opts.mode ~= nil and opts.mode ~= 'alive' and opts.mode ~= 'all' and
+        opts.mode ~= 'ro' and opts.mode ~= 'rw' then
+        local msg = 'Incorrect filter mode "%s", allowed values: '..
```

Comment:
It's suggested to rename the `alive` to `any`.

Reply:
Done.

Reply:
I guess, `all` to `any`. OK.